guides/source/action_cable_overview.md

```diff
@@ -0,0 +1,541 @@
+```
+
+By default, Action Cable allows all requests from localhost:3000 when running in the development environment.
+
+### Other Configurations
+
+The other common option to configure is the log tags applied to the per-connection logger. Here's close to what we're using in Basecamp:
+
+```ruby
+ActionCable.server.config.log_tags = [
+  -> request { request.env['bc.account_id'] || "no-account" },
+  :action_cable,
+  -> request { request.uuid }
+]
+```
+
+Your WebSocket URL might change between environments. If you host your production server via https, you will need to use the wss scheme
+for your ActionCable server, but development might remain http and use the ws scheme. You might use localhost in development and your
+domain in production. In any case, to vary the WebSocket URL between environments, add the following configuration to each environment:
+
+```ruby
+config.action_cable.url = "ws://example.com:28080"
+```
+
+Then add the following line to your layout before your JavaScript tag:
+
+```erb
+<%= action_cable_meta_tag %>
+```
+
+And finally, create your consumer like so:
+
+```coffeescript
+App.cable = Cable.createConsumer()
+```
+
+For a full list of all configuration options, see the `ActionCable::Server::Configuration` class.
+
+Also note that your server must provide at least the same number of database connections as you have workers. The default worker pool is set to 100, so that means you have to make at least that available. You can change that in `config/database.yml` through the `pool` attribute.
+
+## Running the cable server
+
+### Standalone
+The cable server(s) can be separated from your normal application server. It's still a Rack application, but it is its own Rack
+application. The recommended basic setup is as follows:
+
+```ruby
+# cable/config.ru
+require ::File.expand_path('../../config/environment', __FILE__)
+Rails.application.eager_load!
+
+require 'action_cable/process/logging'
+
+run ActionCable.server
+```
+
+Then you start the server using a binstub in bin/cable ala:
+```
+#!/bin/bash
+bundle exec puma -p 28080 cable/config.ru
+```
+
+The above will start a cable server on port 28080. Remember to point your client-side setup against that using something like:
+`App.cable = Cable.createConsumer("ws://basecamp.dev:28080")`.
+
+### In app
+
+If you are using a threaded server like Puma or Thin, the current implementation of ActionCable can run side-along with your Rails application. For example, to listen for WebSocket requests on `/websocket`, match requests on that path:
+
+```ruby
+# config/routes.rb
+Example::Application.routes.draw do
+  match "/websocket", to: ActionCable.server, via: [:get, :post]
+end
+```
+
+You can use `App.cable = Cable.createConsumer("/websocket")` to connect to the cable server.
+
+For every instance of your server you create and for every worker your server spawns, you will also have a new instance of ActionCable, but the use of Redis keeps messages synced across connections.
```

Comment:
Should move In App above Standalone as that's the recommended way to start out with ActionCable.

Also, a logistical question: what is the performance difference between running the specific config.ru file and running it through the total rails server. I'm assuming there's a bit of handshake time between the two since one travels through the middleware and the other goes straight to the websocket, but once connected is it any different? Since the Rails application is being eager loaded, the memory footprint would be nearly identical?

If the performance isn't drastically different, would it be simpler to just spawn new Rails servers dedicated to websockets rather than dealing with a modified config.ru file and task-specific deploy?